content/_code-samples/airgapped-wallet/py/airgapped-wallet-TEST.py

```diff
@@ -0,0 +1,228 @@
+import os
+import base64
+import qrcode
+import platform
+from PIL import Image
+from pathlib import Path, PureWindowsPath, PurePath
+from cryptography.fernet import Fernet
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.kdf.pbkdf2 import PBKDF2HMAC
+from xrpl import wallet
+from xrpl.core import keypairs
+from xrpl.utils import xrp_to_drops
+from xrpl.models.transactions import Payment
+from xrpl.transaction import safe_sign_transaction
+
+
+def create_wallet(silent: False):
+    """
+    Generates a keypair
+    """
+    if not silent:
+        print("1. Generating seed...")
+        seed = keypairs.generate_seed()
+
+        print("2. Deriving keypair from seed...")
+        pub, priv = keypairs.derive_keypair(seed)
+
+        print("3. Deriving classic addresses from keypair..\n")
+        address = keypairs.derive_classic_address(pub)
+        private = keypairs.derive_classic_address(priv)
+
+        return address, private, seed
+    if silent:
+        seed = keypairs.generate_seed()
+        pub, priv = keypairs.derive_keypair(seed)
+        address = keypairs.derive_classic_address(pub)
+        private = keypairs.derive_classic_address(priv)
+        return address, private, seed
+
+
+def sign_transaction(_xrp_amount, _destination, _ledger_seq, _wallet_seq, password):
+    """
+    Signs transaction and returns signed transaction blob in QR code
+    """
+    print("1. Retrieving encrypted private key and salt...")
+    with open(get_path("/WalletTEST/private.txt"), "r") as f:
+        _seed = f.read()
+        _seed = bytes.fromhex(_seed)
+
+    with open(get_path("/WalletTEST/salt.txt"), "rb") as f:
+        salt = f.read()
+
+    print("2. Initializing key...")
+    # Line 49-58: initialize key
```

Comment:
This comment seems out of date, and the print explains it anyway so i think you can just delete it.
```suggestion
```